app/main.py

```diff
@@ -2,0 +1,15 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if parts[0] != "cp" or len(parts) != 3:
+        return
+    source_file, destination_file = parts[1], parts[2]
```

Comment:
Consider using more descriptive variable names, such as `source_file_name` and `destination_file_name`, to improve code readability as suggested in the checklist.